### ISSUE TITLE
argument

### ISSUE THREAD

**Issue by vickersmith** [pangwong/pytorch-multi-label-classifier#5]
hi,i run the multi_label_classifier.py and  met the following error:
usage: multi_label_classifier.py [-h] --dir DIR --name NAME --mode MODE
                                 --model MODEL [--load_size LOAD_SIZE]
                                 [--input_size INPUT_SIZE] [--ratio RATIO]
                                 [--batch_size BATCH_SIZE] [--shuffle]
                                 [--flip] [--region]
                                 [--load_thread LOAD_THREAD] [--crop CROP]
                                 [--gray] [--gpu_ids GPU_IDS]
                                 [--box_ratio BOX_RATIO]
                                 [--box_scale BOX_SCALE]
                                 [--input_channel INPUT_CHANNEL] [--mean MEAN]
                                 [--std STD] [--padding]
                                 [--checkpoint_name CHECKPOINT_NAME]
                                 [--pretrain]
                                 [--validate_ratio VALIDATE_RATIO]
                                 [--sum_epoch SUM_EPOCH]
                                 [--save_epoch_freq SAVE_EPOCH_FREQ]
                                 [--save_batch_iter_freq SAVE_BATCH_ITER_FREQ]
                                 [--lr LR] [--gamma GAMMA]
                                 [--lr_mult_w LR_MULT_W]
                                 [--lr_mult_b LR_MULT_B]
                                 [--lr_policy LR_POLICY]
                                 [--lr_decay_in_epoch LR_DECAY_IN_EPOCH]
                                 [--momentum MOMENTUM]
                                 [--weight_decay WEIGHT_DECAY]
                                 [--loss_weight LOSS_WEIGHT] [--top_k TOP_K]
                                 [--score_thres SCORE_THRES]
                                 [--label_file LABEL_FILE]
                                 [--classify_dir CLASSIFY_DIR]
                                 [--display_winsize DISPLAY_WINSIZE]
                                 [--display_id DISPLAY_ID]
                                 [--display_port DISPLAY_PORT]
                                 [--image_ncols IMAGE_NCOLS] [--html]
                                 [--update_html_freq UPDATE_HTML_FREQ]
                                 [--display_train_freq DISPLAY_TRAIN_FREQ]
                                 [--display_validate_freq DISPLAY_VALIDATE_FREQ]
                                 [--display_data_freq DISPLAY_DATA_FREQ]
                                 [--display_image_ratio DISPLAY_IMAGE_RATIO]
multi_label_classifier.py: error: argument --dir is required
Process finished with exit code 2

i debuged,i found the code in multi_label_classifier.py
    opt = op.parse() 

i debuged step by step ,i found this line can't return a value''opt'" ,very strange.

**Comment by pangwong** [comment]
Some of options are required and some are optional. You will need to specify those required options.

**Comment by vickersmith** [comment]
I have set those required options,e.g. dir,name,but the error still occur.When i change those relevant code "required=True" to "type=str" ,it works.
